magento/graphql-ce
# Transform quote address country code to uppercase

### Description (*)
Country IDs are uppercase and storing `country_id` as lowercase for quote
address objects can cause undesired behavior when the Directory module determines
if a region is allowed for a given country.

Specifically this check fails when fetching the region code for a quote
address:
https://github.com/magento/graphql-ce/blob/a2fc3b98bf52ff7741579d16042b9c6376ecebfc/app/code/Magento/Customer/Model/Address/AbstractAddress.php#L400

### Fixed Issues (if relevant)
1. magento/graphql-ce#658

### Manual testing scenarios (*)
1. Create empty cart
2. Add product to cart
3. Set shipping address on cart meeting criteria:
  - `country_code` must NOT be uppercase
  - `country_code` must be have predefined regions within the application `directory_country_region`
4. Attempt setting shipping method on cart with invalid carrier or method
5. Query cart to include `shipping_addresses { region { code label } }`
6. Region code returned should match set shipping address and not be `null`

### Contribution checklist (*)
